virtwho/config.py

```diff
@@ -545,23 +547,24 @@ def _pre_validate(self):
         Steps necessary to do before evaluation
         """
         if len(self._unvalidated_keys) == 0:
-            self.validation_messages.append(('warning', 'No values provided in: %s' % self.name))
+            if not self._values:
+                self.validation_messages.append(('warning', 'No values provided in: %s' % self.name))
         else:
             for default_key in self.defaults.keys():
-                if default_key not in self._unvalidated_keys:
+                if default_key not in self._unvalidated_keys and default_key not in self._restricted and self.defaults[default_key] is not None:
                     self.validation_messages.append(
                         (
                             'warning',
-                            'Value for "%s" not set in: %s, using default: %s' %
-                            (default_key, self.name, self.defaults[default_key])
+                            'Value for "%s" not set, using default: %s' %
+                            (default_key, self.defaults[default_key])
```

Comment:
Information about section was IMHO useful in warning message. Why is it removed now?

Reply:
Ignore my previous comment. I noticed now that is reported in different way:
```
global: Value for "interval" not set, using default: 3600
```
What about this formating:
```
[global]: Value for "interval" not set, using default: 3600
```

It would be IMHO more obvious that the global is section.